src/core/quantum_circuit.jl

```diff
@@ -267,8 +267,10 @@ yield the same output for any input, up to a global phase.
 Circuits with different ordering of gates that apply on different 
 targets can also be equivalent. 
 !!! note 
-    If there are `Readouts` are present in either `QuantumCircuit`, 
-    they are ignored by `compare_circuit`.
+    If there are `Readouts` are present on either `QuantumCircuit`, 
```

Comment:
Just thinking out loud, should we write `Readout`s or `Readouts`?

Reply:
I find the former looks like a typo

Reply:
It does look odd, we'll then stick with `Readouts`.